test/system/012-manifest.bats

```diff
@@ -143,6 +143,29 @@ EOF
     validate_instance_compression "2" "$list" "amd64" "zstd"
     validate_instance_compression "3" "$list" "arm64" "zstd"
 
+    smanifest="$list"
+    run_podman manifest inspect --authfile=$authfile --tls-verify=false \
+        $manifestpushed
+
+    objects=$(for obj in 0 1 2 3; do echo \
+          ".manifests[$obj].annotations" \
+          ".manifests[$obj].digest" \
+          ".manifests[$obj].platform.architecture" \
+          ".manifests[$obj].platform.os" \
+          ".manifests[$obj].mediaType" \
+          ".manifests[$obj].size" \
+          ; done)
+    for object in \
+        $objects \
+        '.schemaVersion' \
+        '.mediaType' \
+         ; do
+        skopeoObj=$(jq -r "$object" <<<"$smanifest")
+        podmanObj=$(jq -r "$object" <<<"$output")
+        assert "$skopeoObj" != "$smanifest" "\"$object\" does not exist"
+        assert "$skopeoObj" == "$podmanObj" "\"$object\" does not match"
```

Comment:
Suggestion for the diagnostic message: "skopeo-inspect $object does not match podman". Slight (very slight) preference for reversing the terms, `assert "$podmanObj" == "$skopeoObj" "podman-inspect $object does not match skopeo"`, because podman-inspect seems to be the one that is being tested, but perhaps I'm misreading the test.